src/lib.rs

```diff
@@ -133,3 +135,10 @@ pub type Matrix4D<T> = Transform3D<T>;
 #[deprecated]
 pub type TypedMatrix4D<T, Src, Dst> = TypedTransform3D<T, Src, Dst>;
 
+fn min<T: PartialOrd>(x: T, y: T) -> T {
```

Comment:
Yeah, that's a more general rust std sadness topic. They were already there, I just moved them up so that Point::min/max can use that instead of depending on the Float trait.